[Sidecar] Misc Search Test Fixes

joshchong opened the pull request:
Hey @tinatian @vicnate5, this should resolve several search related test failures that were happening on https://github.com/shuyangzhou/liferay-portal/pull/8736. There were several other failures on that PR that were happening due to the temporary test setups on that branch. We'll wait until the Portal build development for sidecar is complete and we don't have to rely on those temp setups before we start testing things again.

petershin committed:
LPS-106592 Create Elasticsearch sidecar zip

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

petershin committed:
LPS-106592 Remove unneeded files

petershin committed:
LPS-106592 Update destination directory

petershin committed:
LPS-106592 Update task names

tinatian committed:
LPS-106592 Make it consistent with client version defined in portal-search-elasticsearch7-impl

tinatian committed:
LPS-103577 portal-search-elasticsearch7-api: buildlang

tinatian committed:
LPS-103577 portal-search-elasticsearch7-api: add new properties to ElasticsearchConfiguration for sidecar process

tinatian committed:
LPS-106592 Copy to liferay home directly instead of a zip file in tools, we have removed bash files in bin, it can not be started accidentally.

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl: add support to start sidecar by default in embedded mode, disable old embedded mode impl for now, will clean it up later.

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl: modify class to allow elasticsearch to run as root user

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl: modify class to avoid dumping keystore to file system, this is needed by IBM JDK cause it does not support empty password which is hard coded by elasticsearch.

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl: disable deprecation warning for sidecar process

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl: clustering-add support to clean up persisted cluster data

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl: add warning message, do not use sidecar in production

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl: add support to launch a separate Elasticsearch JVM as a sidecar JVM

michaelhashimoto committed:
LRCI-995 Remove unneeded put since it should already exist within PathSpiraArtifact

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl:clustering- enable "index.auto_expand_replicas" in default index setting

tinatian committed:
LPS-103577 Enable "index.auto_expand_replicas" for workflow dxp bundle

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl: clustering-add master node to voting exclusions to shutdown master node gracefully, this is only needed when there are two nodes in cluster

Commit:
LRQA-55721 Remove duplicate locators

Commit:
LRQA-55721 Remove duplicate steps to select WC

Commit:
LRQA-55721 Remove duplicate steps to configure AP

Commit:
LRQA-55721 Remove abandoned parameter

Commit:
LRQA-55721 Use clickNoMouseOver to close sidebar

Commit:
LRQA-55721 Remove unnecessary steps

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl: clustering-sikp restart sidecar when there is a network issue

michaelhashimoto committed:
LRCI-995 Add retrieval methods for spira test cases

tinatian committed:
LPS-103577 portal-search-elasticsearch7-impl: clustering-restart sidecar when nodes used to initialize it are removed

michaelhashimoto committed:
LRCI-995 Move methods that can be used in other similar objects to PathSpiraArtifact

michaelhashimoto committed:
LRCI-995 Add methods to create and delete test case folders

michaelhashimoto committed:
LRCI-995 Add methods to create and delete test cases

Commit:
LRQA-55721 Modify locator to make more stable

Commit:
LRQA-55721 Update locator for fragment drop target

liuxingye committed:
LRQA-55782 Remove the ignore property

liuxingye committed:
LRQA-55782 Update the name of breadcrumb

veroglez committed:
LPS-108663 - Modifying searchForm by Clay component

wincent committed:
LPS-108723 Fix misspelled variable name

This is an internal variable, and was misspelled consistently
everywhere, so no actual bug here.

Just cleaning this up because I had to look at this code while verifying
that the tests still worked as a result of the changes made in
LPS-108723.

wincent committed:
LPS-108723 Remove unnecessary Jest mocks

The new version of liferay-npm-scripts sets up some additional useful
mocks by default, allowing us to remove or otherwise simplify the set-up
code in a number of places.

wincent committed:
LPS-108723 Update liferay-npm-scripts to v25.1.0

This expands our default mock environment for use in Jest, which means
we can strip out or simplify config in a bunch of places.

sandrodw3 committed:
LPS-108663 Remove unnecessary small property

wincent committed:
LPS-108317 Fix functions missing in master bundle that is built on win10 OS

Brings in this bundler bug fix:

https://github.com/liferay/liferay-js-toolkit/pull/510

veroglez committed:
LPS-108734 - Modifying topper bar style

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LRCI-995 prep next

joshchong committed:
LPS-103577 Do not merge: temp changes to run ci test

joshchong committed:
LRQA-55521 Update Elasticsearch 7 ports

joshchong committed:
add back default trigger files for testing

tinatian commented:
Close as we are not going to enable sidecar for now
